src/models/RatesByAssetSymbol.js

```diff
@@ -0,0 +1,36 @@
+// @flow
+/*
+    Pillar Wallet: the personal data locker
```

Comment:
Consider renaming file to `Rates.js` as we will probably put different rate-related models here.

Reply:
Yeah, the rename was picked due refactor.